include/zephyr/linker/irq-vector-table-section.ld

```diff
@@ -2,3 +2,10 @@
 
 . = ALIGN(CONFIG_ARCH_IRQ_VECTOR_TABLE_ALIGN);
 KEEP(*(_IRQ_VECTOR_TABLE_SECTION_SYMS))
+
+/*
+ * Some ARM platforms require this symbol to be placed after the IRQ vector
```

Comment:
Is it linked to `CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP`?
If yes, it might be worse stating it.

Reply:
No idea. @ABOSTM is that linked to `CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP`?

Reply:
Only effect of `CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP` is to disable `SW_VECTOR_RELAY_CLIENT`, which will trigger following code to be used:
```
size_t vector_size = (size_t)_vector_end - (size_t)_vector_start;
```
So it seems linked indeed, but this is all F0 specific anyway. So it doesn't add any information. Let's keep it the way you wrote it.

Reply:
> No idea. @ABOSTM is that linked to `CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP`?

:smiley: @erwango told me about this CONFIG switch yesterday.

That said, my understanding is that this is not directly linked to the feature CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP, nevertheless for this feature we make use of symbol `_vector_end` .
But this could have been used for any other purpose.